include/openthread/link.h

```diff
@@ -52,7 +52,6 @@ extern "C" {
  * @{
  *
  */
-#define OT_US_PER_TEN_SYMBOLS 160 ///< The microseconds per 10 symbols.
```

Comment:
Suggest maybe keeping this one as is?

In general, we should try to keep definitions in public or platform headers the same as much as we can (to have backward compatibility with code that may be using OT or implementations of platform layer). 

We can change a public API or definition if we have a good justification for the change (but better to limit such changes).

Reply:
Make sense to me. Let's keep all these existing definitions for compatibility. In this PR I just added the definition of `SYMBOL_RATE`.

Actually the major intention of this PR is to add explanation for how 160 comes from. Even I forgot this for a few times...